docs/developer-guide/syntax-reference.rst

```diff
@@ -1362,6 +1362,84 @@ Scalar functions
+|                        |                                                                           | URL as a single string, see ``URL_EXTRACT_QUERY.``|
+|                        |                                                                           |                                                   |
+|                        |                                                                           | Input: ``http://test.com?a=foo&b=bar``, `a`       |
+|                        |                                                                           | Output: ``foo``                                   |
+|                        |                                                                           | Input: ``http://test.com?a=foo&b=bar``, `b`       |
+|                        |                                                                           | Output: ``bar``                                   |
++------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| URL_EXTRACT_PATH       |  ``URL_EXTRACT_PATH(url)``                                                | Extracts the path from ``url``.                   |
+|                        |                                                                           | Returns NULL if ``url`` is not a valid URI but    |
+|                        |                                                                           | returns "" if the path is empty.                  |
+|                        |                                                                           |                                                   |
+|                        |                                                                           | Input: ``http://test.com/path/to#a``              |
+|                        |                                                                           | Output: ``path/to``                               |
++------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| URL_EXTRACT_PORT       |  ``URL_EXTRACT_PORT(url)``                                                | Extract the port number from ``url``.             |
+|                        |                                                                           | Returns NULL if ``url`` is not a valid URI or does|
+|                        |                                                                           | not contain an explicit port number.              |
+|                        |                                                                           |                                                   |
+|                        |                                                                           | Input: ``http://test@localhost:8080``             |
```

Comment:
maybe the addition of the user name is detracting from the example? It's not that common a pattern.  Maybe just use `http://someServer:8080/some/path`?